### PR TITLE
Krishnan | Added the feature to sort Patient Search Results alphabeti…

### PR DESCRIPTION
**Pull request by joaomachiana**
…cally/numerically etc

### CODE REVIEW

File: ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+angular.module("naturalSorting", [])
+
+// The core natural service
+.factory("naturalService", ["$locale", function ($locale) {
```
Comment (nthfloor) [inline ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: @krishnanspace shouldn't we be adding unit tests for this service?

I release this was implemented as a high priority, but is there a plan or a note to get back to this at some point?

Comment (krishnanspace) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: @nthfloor The above service is actually an external library I had used for Natural Sort.

Comment (nthfloor) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: What do you mean by an external library? From what I can see this is a custom service you are adding to be used by the Bahmni app. So what is making this external and why does that mean we should not unit test it here?

Comment (krishnanspace) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: We should be unit testing it. Thats for sure.What I meant was that I had not written the service

Comment (nthfloor) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: ok, and is this something is being contributed back to the community?

Comment (krishnanspace) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: Yes, I can see this as a contribution.

Comment (nthfloor) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: Ok then I will create a new ticket to add some unit tests and work on contributing this back to the community. Sound good?

Comment (krishnanspace) [reply ui/app/common/registrationNaturalFilter/registrationNaturalFilter.js]: Yup!

**Pull request merged**